### PR TITLE
Refactor/v1-173 - Deleted all importants

### PR DESCRIPTION
**Pull request by ostapdans99**
- Frontend

1. Deleted all importants

### CODE REVIEW

File: frontend/src/components/Layout/Menu/styled.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import styled from 'styled-components';
-import { makeStyles } from '@material-ui/core';
-import { ListItemButton, ListItemText } from '@mui/material';
 import { ReactFragment } from 'react';
+import { styled, ListItemButton, ListItemText } from '@mui/material';
+import { makeStyles } from '@material-ui/core';
```
Comment (irina-sinitsyna) [inline frontend/src/components/Layout/Menu/styled.ts:3]: please use import from mui instead

File: frontend/src/components/Layout/MobileMenu/styled.ts [hunk 2]
```diff
@@ -1,8 +1,7 @@
-import styled from 'styled-components';
 import { ReactFragment } from 'react';
 import { Link } from 'react-router-dom';
+import { styled, ListItemButton, Backdrop } from '@mui/material';
 import { makeStyles } from '@material-ui/core';
```
Comment (irina-sinitsyna) [inline frontend/src/components/Layout/MobileMenu/styled.ts:4]: mui

File: frontend/src/components/Layout/styled.ts [hunk 3]
```diff
@@ -1,13 +1,12 @@
+import { styled, Grid } from '@mui/material';
 import { makeStyles } from '@material-ui/core';
```
Comment (irina-sinitsyna) [inline frontend/src/components/Layout/styled.ts:2]: mui

File: frontend/src/components/Layout/styled.ts [hunk 4]
```diff
@@ -35,18 +34,18 @@ export const PageWrapper = styled(Grid)({
...
-    width: 'calc(100% - 275px)!important',
+    width: 'calc(100% - 275px)',
     [theme.breakpoints.down('md')]: {
-      width: '100%!important',
+      width: '100%',
     },
   },
   showPageWrapper: {
-    width: `calc(100% - 129px)!important`,
+    width: `calc(100% - 129px)`,
```
Comment (irina-sinitsyna) [inline frontend/src/components/Layout/styled.ts:46]:
```suggestion
    width: 'calc(100% - 129px)',
```

File: frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/styled.ts [hunk 5]
```diff
@@ -30,17 +29,17 @@ export const EmployeeButtonGroup = styled(ButtonGroup)({
 });
 
 export const SkillsAndCoursesButton = styled(ToggleButton)<ToggleButtonProps>(({ isOpened }) => ({
-  backgroundColor: '#ffffff !important',
-  color: '#000000 !important',
-  fontSize: '16px !important',
-  fontWeight: '400 !important',
-  width: '135px !important',
+  backgroundColor: '#ffffff',
+  color: '#000000',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/styled.ts:33]: please use uppercase or palette

File: frontend/src/pages/NotFound/styled.ts [hunk 6]
```diff
@@ -43,18 +41,18 @@ export const UnderErrorText = styled(Typography)({
...
-    fontSize: '40px !important',
-    lineHeight: '40px !important',
+    fontSize: '40px',
+    lineHeight: '40px',
   },
   [theme.breakpoints.up('lg')]: {
-    fontSize: '60px !important',
-    lineHeight: '60pxpx !important',
+    fontSize: '60px',
+    lineHeight: '60pxpx',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/NotFound/styled.ts:56]:
```suggestion
    lineHeight: '60px',
```

File: frontend/src/components/Layout/Menu/styled.ts (hunk 1 above)
Comment (ostapdans99) [reply frontend/src/components/Layout/Menu/styled.ts:3]: For example: https://mui.com/material-ui/api/list-item-button/

Imports: 
image

File: frontend/src/components/Layout/MobileMenu/styled.ts (hunk 2 above)
Comment (ostapdans99) [reply frontend/src/components/Layout/MobileMenu/styled.ts:4]: same

File: frontend/src/components/Layout/styled.ts (hunk 3 above)
Comment (ostapdans99) [reply frontend/src/components/Layout/styled.ts:2]: same

File: frontend/src/pages/NotFound/styled.ts (hunk 6 above)
Comment (ostapdans99) [reply frontend/src/pages/NotFound/styled.ts:56]: done

File: frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/styled.ts (hunk 5 above)
Comment (ostapdans99) [reply frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/styled.ts:33]: done

File: frontend/src/components/Layout/styled.ts (hunk 4 above)
Comment (ostapdans99) [reply frontend/src/components/Layout/styled.ts:46]: done

**Review by irina-sinitsyna — APPROVED** [review]

**Pull request merged**